lib/explorer/series.ex

```diff
@@ -841,6 +841,46 @@ defmodule Explorer.Series do
   def strftime(%Series{dtype: dtype}, _format_string),
     do: dtype_error("strftime/2", dtype, [:datetime])
 
+  @doc """
+  Clip (or clamp) the values in a series.
+
+  Values that fall outside of the interval defined by the `min` and `max`
+  bounds are clipped to the bounds.
+
+  ## Supported dtypes
+
+    * `:integer`
+    * `:float`
+
+  Clipping other dtypes are possible using `select/3`.
+
+  ## Examples
+
+      iex> s = Explorer.Series.from_list([-50, 5, nil, 50])
+      iex> Explorer.Series.clip(s, 1, 10)
+      #Explorer.Series<
+        Polars[4]
+        integer [1, 5, nil, 10]
+      >
+  """
+  @doc type: :element_wise
+  @spec clip(series :: Series.t(), min :: number(), max :: number()) :: Series.t()
+  def clip(%Series{dtype: dtype} = series, min, max) when is_numeric_dtype(dtype) do
+    if K.and(is_number(min), is_number(max)) do
+      min = if is_integer(min), do: min / 1.0, else: min
+      max = if is_integer(max), do: max / 1.0, else: max
+      apply_series(series, :clip, [min, max])
```

Comment:
This should do it:

```suggestion
      apply_series(series, :clip, [min * 1.0, max * 1.0])
```

But keep in mind that, for large integers, this will lead to loss of precision. If possible we would pass both integers and floats, but if Polars only accepts floats, this is fine. :)